pkg/reconciler/scheduling/placement/controller.go

```diff
@@ -84,6 +81,9 @@ func NewController(
 
 		placementLister:  placementInformer.Lister(),
 		placementIndexer: placementInformer.Informer().GetIndexer(),
+
+		edgePlacementLister:  edgePlacementInformer.Lister(),
+		edgePlacementIndexer: edgePlacementInformer.Informer().GetIndexer(),
```

Comment:
As you can tell by the stuttering ("...Informer.Informer()"),"edgePlacementInformer" is _not_ actually an _informer_, it is something prior to that.

Reply:
Right, but let's to keep the style consistent with kcp's (so many) controllers? It works well.

Reply:
The style in the kcp code is not self-consistent, and does not work well in my opinion --- it does not always make it clear what is an all-cluster object vs what is a cluster-specific object.

I can tolerate not stuffing all relevant facts into variable names.  We do not want each variable name to be an essay.  But since kcp introduces the concept of all-cluster objects but also retains cluster-specific objects, it is an issue that is of particular sensitivity.

Reply:
> We should probably have a general discussion of this and other style issues.


Yes, I agree, it is good to bring everyone to the same page in this early stage.